### PR TITLE
Fixed some regression issues for the add from trakt trending and add …

### PR DESCRIPTION
**Pull request by p0psicles**
…from IMDB popular shows.

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked
- [x] Give a description on what the PR is for.
- [ ] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

File: gui/slick/js/core.js [hunk 1]
```diff
@@ -3263,46 +3263,46 @@ var SICKRAGE = {
...
+		        var configure_show_options = $('#configure_show_options').prop('checked');
                 var indexer = $(this).data('indexer');
-                var indexerId = $(this).data('indexer_id');
-                var showName = $(this).data('show_name');
-                var defaultStatus = $('#statusSelect').val();
-                var qualityPreset = $('#qualityPreset').val();
-                var anyQualities = anyQualArray.join(',');
-                var bestQualities = bestQualArray.join(',');
-                var defaultFlattenFolders = $('#flatten_folders').prop('checked');
+                var indexer_id = $(this).data('indexer_id');
```
Comment (OmgImAlexis) [inline gui/slick/js/core.js:3271]: :-1: this is wrong. These are meant to be camelcase. All of the JS vars are.

**Comment by OmgImAlexis** [comment]
@p0psicles this is the kind of stuff that I said is going to have to be fixed before we merge. If you keep writing js like the other languages you use then it's just going to cause more work for the devs that need to fix it before it gets added to dev.

**Comment by p0psicles** [comment]
Really? I write the js code in CamelCase but then i'm not compliant to python syntax. So I change the JS vars to be compliant. My opinion here is that by adding this as an exception, i'm not breaking anything. It's still readable and i'm PEP8 compliant.

@medariox @miigotu @labrys what do you guys think?

**Comment by labrys** [comment]
@p0psicles js code should be js style, python code should be python style, and PEP8 has nothing to do with js code.  Where the two meet there are methods to make things play nice, but they should be used consistently throughout and not just choose a method that suits a dev at the time.  Consistency is important.

**Comment by OmgImAlexis** [comment]
@p0psicles you should be using single quotes around the snake_case instead of just turning off lint warnings. We all spent a LONG time converting this to use linters for everything to stop things like this happening, that's why travis actually fails if it hits a lint error/warning.

**Pull request merged**